pyvo/dal/params.py

```diff
@@ -414,9 +418,13 @@ def get_dal_format(self, val):
             raise ValueError('Invalid interval: min({}) > max({})'.format(
                 low, high))
         if self._unit:
+            if isinstance(low, str):
```

Comment:
Here, again, converting to a unit seems wrong. One can just do `low = Quantity(low)`, assuming that the string contains a unit too (otherwise, it is going to be dimensionless).